src/main/java/co/cask/gcp/bigquery/BigQuerySource.java

```diff
@@ -164,6 +169,22 @@ public String getInputFormatClassName() {
         return config;
       }
     }));
+
```

Comment:
@albertshau In the lines above this line 157 where we setInput should we use referenceName now?

Reply:
yes